### PR TITLE
DataPusher and XLoader documented in Source docs

### LINKED ISSUE #6269: DataPusher not documented in CKAN source installing document.
**CKAN version**
Master

**Describe the bug**
There is no information about DataPusher in CKAN source installing doc: https://docs.ckan.org/en/2.9/maintaining/installing/install-from-source.html

**Expected behavior**
DataPusher must be documented after Datastore in the CKAN source installing document.

### PR DESCRIPTION
**Pull request by Gauravp-NEC**
Fixes #6269 

### Proposed fixes:
DataStore and XLoader are documented in CKAN source installing document.

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by Gauravp-NEC** [comment]
@kowh-ai , Please review this PR. If it looks good please merge it. Thankyou :)

File: doc/maintaining/installing/install-from-source.rst [hunk 1]
```diff
@@ -277,6 +277,10 @@ Follow the instructions in :doc:`/maintaining/datastore` to create the required
 databases and users, set the right permissions and set the appropriate values
 in your CKAN config file.
 
+After setting up the DataStore, configure DataPusher or XLoader to add data to DataStore.
+To install the DataPusher refer this docs: https://github.com/ckan/datapusher or refer this
+docs: https://github.com/ckan/ckanext-xloader to install the XLoader.
+
 ---------------
```
Comment (kowh-ai) [inline doc/maintaining/installing/install-from-source.rst:284]: Once you have set up the DataStore, you may then wish to configure either the DataPusher _or_ XLoader extensions to add data to the DataStore. To install DataPusher refer to this link: https://github.com/ckan/datapusher or refer to this link: https://github.com/ckan/ckanext-xloader to install XLoader.

**Review by kowh-ai — CHANGES_REQUESTED** [review]
I think this reads a bit better

**Comment by Gauravp-NEC** [comment]
@kowh-ai , Based on your above suggestion I have done the minor rework. Please review it. Thankyou : )

**Comment by Gauravp-NEC** [comment]
@kowh-ai , Based on your above suggestion I have done the rework. Now this PR is ready to merged. Please review it. Thankyou : )

**Review by kowh-ai — APPROVED** [review]
Looks good - thanks

**Comment by Gauravp-NEC** [comment]
@kowh-ai , Thankyou for reviewing this PR. Please merge this PR :)

**Comment by kowh-ai** [comment]
> @kowh-ai , Thankyou for reviewing this PR. Please merge this PR :)

I don't have enough access to merge into the CKAN repo - sorry

**Comment by Gauravp-NEC** [comment]
@amercader , This PR is approved by @kowh-ai. Please merge this PR. Thankyou :)

**Pull request merged**